### DISCUSSION TITLE
一連串錯誤政策 特拉斯失民心

### DISCUSSION THREAD

**Discussion by Floriaand** [1101-GS4719-C/Python-Community-GS4719-C#809] [General]
英國首相特拉斯上任45天後辭職下台，雖然她只擔任了一個多月的首相，但卻在英國政治和經濟上掀起了巨浪，真的算是我有記憶以來英國金融市場動盪的最厲害的一次，上個星期才看到她說不會下台，結果這個星期就辭職了，可能還是因為失了民心吧。

網址：https://udn.com/news/amp/story/7339/6702867?from=udnamp-referralnews_ch2artbottom

**Comment by Chiway1229** [comment]
最近英國的政局確實動盪不安，女王離世，親王年事已大而且首相又無法定民心，希望繼位的首相能將英國局勢力挽狂瀾

**Comment by CPCpyt** [comment]
當前全球總體經濟呈現不穩定且下行的趨勢，
全球經濟就如同不會調轉的火車頭，
以一個國家來說要對抗這樣的趨勢實在困難。
